metricbeat/module/mysql/overview/overview.go

```diff
@@ -0,0 +1,95 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+/*
+Package status fetches MySQL server status metrics.
+
+For more information on the query it uses, see:
+http://dev.mysql.com/doc/refman/5.7/en/show-status.html
+*/
+package overview
+
+import (
+	"context"
+	"github.com/elastic/beats/v7/metricbeat/helper/sql"
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	"github.com/elastic/beats/v7/metricbeat/module/mysql"
+	"github.com/pkg/errors"
```

Comment:
nit: import order

Reply:
I finally removed the `overview` metricset for a follow up PR. Thanks!